docsite/rst/developing_plugins.rst

```diff
@@ -56,12 +72,20 @@ If you want more Jinja2 filters available in a Jinja2 template (filters like to_
 
 Jump into `lib/ansible/plugins/filter <https://github.com/ansible/ansible/tree/devel/lib/ansible/plugins/filter>`_ for details.
 
+Test Plugins
+------------
+
+If you want more Jinja2 tests available in a Jinja2 template (tests like 'even' and 'odd' are provided by default), they can be extended by writing a test plugin.  Most of the time, when someone comes up with an idea for a new test they would like to make available in a playbook, we'll just include them in 'core.py' instead.
```

Comment:
...we'll just include them in 'core.py' instead.
suggest: they will be included in 'core.py' instead.